### ISSUE TITLE
[BUG] Camera images have wrong frame_id

### ISSUE THREAD

**Issue by peci1** [luxonis/depthai-ros#162]
**Describe the bug**
Run driver with `roslaunch --screen depthai_examples rgb_stereo_node.launch`. Then `rostopic echo /rgb_stereo_publisher/color/camera_info`. The `frame_id` is `dai_rgb_camera_optical_frame`. Similarly for depth images. However, this frame is not defined anywhere:

```
$ rosrun tf tf_monitor
RESULTS: for all Frames

Frames:
Frame: oak-d_frame published by unknown_publisher(static) Average Delay: 0 Max Delay: 0
Frame: oak_imu_frame published by unknown_publisher(static) Average Delay: 0 Max Delay: 0
Frame: oak_left_camera_frame published by unknown_publisher(static) Average Delay: 0 Max Delay: 0
Frame: oak_left_camera_optical_frame published by unknown_publisher(static) Average Delay: 0 Max Delay: 0
Frame: oak_model_origin published by unknown_publisher(static) Average Delay: 0 Max Delay: 0
Frame: oak_rgb_camera_frame published by unknown_publisher(static) Average Delay: 0 Max Delay: 0
Frame: oak_rgb_camera_optical_frame published by unknown_publisher(static) Average Delay: 0 Max Delay: 0
Frame: oak_right_camera_frame published by unknown_publisher(static) Average Delay: 0 Max Delay: 0
Frame: oak_right_camera_optical_frame published by unknown_publisher(static) Average Delay: 0 Max Delay: 0
```

**Minimal Reproducible Example**
See above.

**Expected behavior**
All camera frame_ids should be defined either in the URDF or by static TF publishers.

**Attach system log**
[log_system_information.zip](https://github.com/luxonis/depthai-ros/files/9975716/log_system_information.zip)
 - Which ROS version are you using? Melodic

**Additional context**

It seems like `tf_prefix` is not somehow propagating inside rgb_stereo_node.

**Comment by Serafadam** [comment]
Hi, `rgb_stereo_node` will probably be soon deprecated, for now it's better to use `stereo_intertial_node` since it's more developed

**Comment by peci1** [comment]
You're right, `stereo_inertial_node` works perfectly (up to a small glitch fixed by #198).

**Comment by Serafadam** [comment]
Hey @peci1 wanted to notify that there is a brand new driver  that has more functionalities that the example `stereo_intertial_node`, I'd be happy to get some feedback on its usability :slightly_smiling_face:  https://github.com/luxonis/depthai-ros/tree/noetic (also available in Humble)
